src/routes.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import Dashboard from './containers/Dashboard'
+import {Login} from './containers/Login'
+import Profile from './containers/Profile'
+import { AuthStore } from './repos/UserRepo'
+import {
+Route,
+Switch,
+Redirect 
+} from 'react-router-dom'
+
+
+const  renderIfAuthenticated = (Component) => props => AuthStore.isAuthenticated ? 
```

Comment:
this method is used by `<PrivateRoute>` below to check the authentication status, and either allow or redirect to the login page , we pass the current location to tell the login component to redirect back to it if login is successful.